source/getting-started/atlas-log-in.txt

```diff
@@ -10,9 +10,10 @@ Log In with Atlas
    :depth: 1
    :class: singlecol
 
-You can sign in to your Atlas account inside Relational Migrator. When 
-you log in with your Atlas account in Relational Migrator, you can 
-access the :ref:`Query Converter <rm-query-converter>` feature.
+You can optionally sign in to your Atlas account inside Relational 
+Migrator. If you log in, you can access the 
+:ref:`Query Converter <rm-query-converter>` feature and cluster names 
```

Comment:
I think it would be clearer to say "you can choose destination clusters from a list when creating migration jobs".